### PR TITLE
[WIP] cinnamon-screensaver-command.py: take into account the org.cinnamon.desktop.lockdown disable-lock-screen boolean value

### PR DESCRIPTION
**Pull request by claudiux**
**Work In Progress.**

This PR is a part of multiple PR. See https://github.com/linuxmint/cinnamon/issues/7592.

### CODE REVIEW

File: src/cinnamon-screensaver-command.py [hunk 1]
```diff
@@ -87,7 +90,10 @@ def perform_action(self):
             else:
                 print(gettext.ngettext ("The screensaver has been active for %d second.\n", "The screensaver has been active for %d seconds.\n", time) % time)
         elif self.action_id == Action.LOCK:
-            self.client.proxy.call_lock_sync(self.message)
+            if not self.settings.get_boolean("disable-lock-screen"):
+                self.client.proxy.call_lock_sync(self.message)
+            else:
+                print(_("Screen lock is not allowed at this time.\n"))
```
Comment (henrykrumb) [inline src/cinnamon-screensaver-command.py]: `gettext.gettext` should be used here instead of the underscore.

Comment (claudiux) [reply src/cinnamon-screensaver-command.py]: I used the underscore, as it is used at lines 83, 85, 89. Maybe should we replace all the underscores by `gettext.gettext`? @clefebvre ?

Comment (claudiux) [reply src/cinnamon-screensaver-command.py]: All suggested replacements are made.

**Comment by claudiux** [comment]
Not enough time at the moment. I will reopen this more or less soon ...

**Pull request closed without merging**